config/install/field.field.paragraph.promotion_card.field_promo_card_display_style.yml

```diff
@@ -16,7 +16,7 @@ required: true
 translatable: false
 default_value:
   -
-    value: standard
+    value: noImage
```

Comment:
Hi @MdNadimHossain, I changed the default value to `noImage` as there is no `standard` value. you are ok with this?